app/Insightly/Repositories/EloquentInsightlyMappingRepository.php

```diff
@@ -24,7 +24,7 @@ public function save(InsightlyMapping $insightlyMapping): void
     public function getById(UuidInterface $id): InsightlyMapping
     {
         /** @var InsightlyMappingModel $insightlyMappingModel */
-        $insightlyMappingModel = InsightlyMappingModel::query()->findOrFail($id);
+        $insightlyMappingModel = InsightlyMappingModel::query()->where('id', $id)->firstOrFail();
```

Comment:
This will have to be refactored when we have both Projects & Opportunities in `insightly_mappings`.

Reply:
I think we will have to add a `getByIdAndType` method and remove `getById`. But that can be done in a later pull request.